src/main/java/duke/task/Todos.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+/** Class to represent the task set by user*/
+public class Todos extends Task {
```

Comment:
Class names should be singular if I am not wrong. Maybe change it to ToDo class instead. Nitpicking here